docs/source/configuration.rst

```diff
@@ -60,14 +60,14 @@ However:
 
 .. code-block:: shell
 
-   tartufo --config tartufo.toml --config myconfig.toml scan-local-repo .
+   tartufo --config tartufo.toml --config myconfig.toml scan-local-repo <target_directory>
 
-will cause ``tartufo`` to read ``tartufo.toml`` in the current directory
+will cause ``tartufo`` to read ``tartufo.toml`` in the target directory
```

Comment:
```suggestion
will cause ``tartufo`` to read ``tartufo.toml`` in the current directory
```

Reply:
Shouldn't this change to `target_directory`?